pkgs/by-name/st/structorizer/package.nix

```diff
@@ -82,24 +56,16 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
 
-    install -d $out/bin $out/share/mime/packages
+    install -d $out/bin $out/share/mime/packages $out/share/applications
 
     install -D ${pname}.jar -t $out/share/java/
       makeWrapper ${jdk11}/bin/java $out/bin/${pname} \
       --add-flags "-jar $out/share/java/${pname}.jar" \
       --prefix _JAVA_OPTIONS " " "-Dawt.useSystemAAFontSettings=gasp" \
       ''${gappsWrapperArgs[@]}
 
-    cat << EOF > $out/share/mime/packages/structorizer.xml
-    <?xml version="1.0" encoding="UTF-8"?>
-    <mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
-      <mime-type type="application/nsd">
-             <comment xml:lang="en">Nassi-Shneiderman diagram</comment>
-             <comment xml:lang="de">Nassi-Shneiderman-Diagramm</comment>
-             <glob pattern="*.nsd"/>
-      </mime-type>
-    </mime-info>
-    EOF
+    cp freedesktop/mime/packages/structorizer.xml $out/share/mime/packages/
+    cp freedesktop/applications/structorizer.desktop $out/share/applications/
 
     cd src/lu/fisch/${pname}/gui
     install -vD icons/000_${pname}.png $out/share/icons/hicolor/16x16/apps/${pname}.png
```

Comment:
That pname matches here is just coincidence and it should not be substituted like this.